kuma/wiki/models.py

```diff
@@ -323,7 +323,8 @@ def get_body_html(self, *args, **kwargs):
         sections_to_hide = ('Quick_Links', 'Subnav')
         doc = parse_content(html)
         for sid in sections_to_hide:
-            doc = doc.replaceSection(sid, '<!-- -->')
+            doc = doc.replaceSection(sid, '')
+            doc = doc.removeSection(sid)
```

Comment:
The tests pass for me without the second line ``doc = doc.removeSection(sid)``. I suspect it isn't doing anything.

Reply:
After discussing this with @stephaniehobson, the issue is that KumaScript injects a ``<section id="Quick_Links>``, not an ``<h3 id="Quick_Links">`` like the test does. Adding a commit that fixes the test fixture to be more accurate and demonstrate the problem that the PR is trying to solve.